api-inference-community/docker_images/speechbrain/app/pipelines/audio_classification.py

```diff
@@ -0,0 +1,37 @@
+from typing import Dict, List
+
+import numpy as np
+import torch
+from app.pipelines import Pipeline
+from speechbrain.pretrained import EncoderClassifier
+
+
+class AudioClassificationPipeline(Pipeline):
+    def __init__(self, model_id: str):
+        self.model = EncoderClassifier.from_hparams(source=model_id)
```

Comment:
Here, as discussed, I wonder if it wouldn't be better to load a class given as an argument to the config.json.

Reply:
Yes, there will be a larger refactoring P of all `speechbrain` to load the class specified the `config.json`.  We want to make an announcement of `audio-classification` today so we're trying to fastrack this PR to have working Inference API and I'll follow up this week with the change to the specified class.